ros/src/tl_detector/tl_detector.py

```diff
@@ -16,6 +16,7 @@
 
 STATE_COUNT_THRESHOLD = 3
 UPDATE_RATE = 10
+
```

Comment:
This shouldn't have been in the commit.  I just had it set to ground truth for testing and added it by mistake.

Reply:
Ok, can you set it back on the next commit to this PR?